### DISCUSSION TITLE
Unable to open folders with symbolic links on Mac

### DISCUSSION THREAD

**Discussion by asifgosla** [phcode-dev/phoenix#1886] [Q&A]
I am using Phoenix code on Mac.  What I found out is that I can't open folders in phcode if they have any symbolic links in them.  Has anyone else seen this and is there a way to get around this?

I can duplicate this problem as follows:
1. Create a top level directory.
2. Create directory and add a single file.
3. Start phcode and click on Open Folder.  
4. Navigate to the test directory. Note that Phcode can successfully open this directory in phcode and the file created in step 1 is visible.
5. Exit phcode
6. Add a symbolic link to another file (or directory) inside the test directory.
7. Start phcode and click on Open Folder.  
8. Navigate to the test directory. Note that Phcode won't open the test directory.

**Comment by abose** [comment]
Thanks for reporting the issue @asifgosla 

Currently, Phoenix code does not support symbolic links. I have added an issue to enable support for symlinks @ https://github.com/phcode-dev/phoenix/issues/1888 .

 But at the very least, it should be able to open folders with symlinks rather than completely bailing out. Let me see if something can be done in that part.

**Reply by asifgosla** [reply]
Thanks!

**Reply by abose** [reply]
@asifgosla i am unable to reproduce this issue, phoenix code was able to open files and directories with symlinks for me. i created the symlinks using `ln -s` command. Is this how you created symlinks?

**Reply by asifgosla** [reply]
Yes I had created symlinks with ln -s.  
Note that I am using MacOS Sequoia.
The Phoenix Code is Version 3.9.4 (20240909.074643).

* Also I can open the files in the directory directly if I drag them in from finder into Phoenix Code, however the directory with the symlinks inside of it doesn't show its contents in the sidebar (even the contents that are regular files and not symlinks).

**Reply by asifgosla** [reply]
@abose I just tried using the web editor at phcode.dv (instead of the Mac desktop app) and found that the web editor doesn't have this problem...it works as expected.
